model_wrappers.py

```diff
@@ -151,18 +149,17 @@ def get_judge_confidence(
 
     def get_debater_argument(
         self,
-        question: str,
-        justify_letter: str,
-        justify_numeric: int,
-        proof_a: str,
-        proof_b: str,
-    ):
+        argument: Argument,
+        cache: Cache
+    ) -> str:
+        if cache[argument] is not None:
+            return cache[argument]
```

Comment:
Should the cache track the formatted rather than unformatted prompt, in case the prompt formatting changes